sample-schemas/demo_schema.erl

```diff
@@ -2,7 +2,7 @@
 
 -include_lib("typerefl/include/types.hrl").
 
--type birthdays() :: [#{m := 1..12, d := 1..31}].
+-type birthdays() :: [map()].
```

Comment:
@zmstone `atom keys -> binary keys` applied. Now we cannot have detailed typing on `birthdays`, as `-type` does not allow binary map keys.

Reply:
ok.
we should probably not encourage objects without a schema anyway.